examples/simple_spi_rw_example/main/simple_spi_rw_example.cpp

```diff
@@ -12,38 +12,36 @@
  * CONDITIONS OF ANY KIND, either express or implied.
 */
 
-#include <iostream>
+#include "sdkconfig.h"
 #include <thread>
 #include "spi_host_cxx.hpp"
 
 using namespace std;
 using namespace idf;
 
-static const GPIONum NSS(23);
 static const uint8_t READ_FLAG = 0x80;
 static const uint8_t MPU9250_WHO_AM_I_REG_ADDR = 0x75;
 
+namespace {
+static const MOSI MOSI_PIN(CONFIG_SPI_MOSI);
+static const MISO MISO_PIN(CONFIG_SPI_MISO);
+static const SCLK SCLK_PIN(CONFIG_SPI_SCLK);
+static const CS     CS_PIN(CONFIG_SPI_CS);
+}
+
 extern "C" void app_main(void)
 {
-    try {
+        SPIMaster master(SPINum(1),
```

Comment:
Nitpick: since try/catch is removed, this block can now be moved left by 4 spaces